### PR TITLE
[+VV] Tools 6 show tableau dashboards

### PR DESCRIPTION
**Pull request by ionut998**
Implement application functionality from [old analytics frontend](https://github.com/reevoo/client_portal-analytics/tree/master/frontend)
- Load user dashboards from Backend and select first dashboard when page loads
- Update dashboard iframe based on the selected navigation item
- Get dashboard token and use it to render the dashboard iframe 
- Implement react-router

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
*If you're new, read the [guidelines]
(https://github.com/reevoo/guidelines/blob/master/pull_requests.md).*

Good code is **tested**, **designed for change**, **easy to understand**
and **able to cope when things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [ ] Description explains what you have done and why.
- [ ] Tests have been written (or you have explained why you don't need
tests).
- [ ] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] The code is well-designed and able to cope with change.
- [ ] The code is easy to understand.
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] Tests are green on CI (or pulled/tested on your machine).

File: app/js/app.jsx [hunk 1]
```diff
@@ -34,20 +39,32 @@ const rvMuiTheme = getMuiTheme({
   },
 })
 
-let store = createStore(analyticsApp)
+// Compose reduxReactRouter with other store enhancers
+const store = compose(
```
Comment (davidsevcik) [inline app/js/app.jsx]: I still don't get this :)

**Comment by davidsevcik** [comment]
LGTMBWTFDIK 👍

File: app/js/app.jsx (hunk 1 above)
Comment (irikuo) [inline app/js/app.jsx]: http://redux.js.org/docs/api/compose.html :)

**Pull request merged**